README.md

```diff
@@ -32,7 +32,6 @@ tft:
         instances: (7)
         server:
           - name: "(8)"
-            persistent: "(9)"
```

Comment:
This is useful for debugging. We should have a way to persist the server.

Reply:
this is now implemented.

Note that there is no auto-detection of the flag. It defaults to FALSE, but is now configurable via the config YAML.

Previously it was always FALSE, but there was a code comment that would indicate that the default should somehow be autodetected. If that's useful, it's a future feature.